config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -81,7 +81,7 @@ presubmits:
         - "./scripts/ci-e2e.sh"
         env:
           - name: GINKGO_FOCUS
-            value: "Workload cluster creation"
+            value: "Creating .* control-plane cluster"
```

Comment:
does this need to have control-plane in there? If not `Creating .* cluster` might be more suited in some cases like "create a managed cluster"

Reply:
No, it doesn't need to have "control-plane," I was just imagining possible accidental matches so I made it as specific as possible. But we can relax it--originally I had `Creating .* cluster`.